Cour-de-cassation/juritj
# feature : add commissioning date to env variable

## Issue description :
The objective is to have commissioning date in env variable to be able to modulate this date in depending on environment

## Describe your changes :
- add commissioning date to env var
- setup a default date if env var is not provided (default date is the production commissioning date)
- some clean on the `computeLabelStatus.ts`

## How to test :
- setup a commissioning date in `.env`
- send some decisions before and after this commissioning date
- check that the batch correctly compute the labelStatus
- do the same test without commissioning date in `.env` file and check that commissioning date is fixed to 2023-12-15

## Checklist before requesting a review
- [x] If it's relevant I added tests. _computeLabelStatus have already relevant tests_
